packages/asset-swapper/src/utils/market_operation_utils/constants.ts

```diff
@@ -131,54 +161,98 @@ export const MAINNET_CURVE_INFOS: { [name: string]: CurveInfo } = {
+        buyQuoteFunctionSelector: CurveFunctionSelectors.None,
+        poolAddress: POOLS.curve_PAX,
+        tokens: [TOKENS.DAI, TOKENS.USDC, TOKENS.USDT, TOKENS.PAX],
+    },
+    Y: {
         exchangeFunctionSelector: CurveFunctionSelectors.exchange_underlying,
         sellQuoteFunctionSelector: CurveFunctionSelectors.get_dy_underlying,
         buyQuoteFunctionSelector: CurveFunctionSelectors.get_dx_underlying,
-        poolAddress: '0x45f783cce6b7ff23b2ab2d70e416cdb7d6055f51',
+        poolAddress: POOLS.curve_y,
         tokens: [TOKENS.DAI, TOKENS.USDC, TOKENS.USDT, TOKENS.TUSD],
     },
-    // Looks like it's dying.
-    DaiUsdcUsdtBusd: {
+    BUSD: {
         exchangeFunctionSelector: CurveFunctionSelectors.exchange_underlying,
         sellQuoteFunctionSelector: CurveFunctionSelectors.get_dy_underlying,
         buyQuoteFunctionSelector: CurveFunctionSelectors.get_dx_underlying,
-        poolAddress: '0x79a8c46dea5ada233abaffd40f3a0a2b1e5a4f27',
-        tokens: [
-            TOKENS.DAI,
-            TOKENS.USDC,
-            TOKENS.USDT,
-            '0x4fabb145d64652a948d72533023f6e7a623c7c53', // bUSD
-        ],
+        poolAddress: POOLS.curve_BUSD,
+        tokens: [TOKENS.DAI, TOKENS.USDC, TOKENS.USDT, TOKENS.BUSD],
     },
-    DaiUsdcUsdtSusd: {
+    sUSD: {
         exchangeFunctionSelector: CurveFunctionSelectors.exchange_underlying,
         sellQuoteFunctionSelector: CurveFunctionSelectors.get_dy_underlying,
         buyQuoteFunctionSelector: CurveFunctionSelectors.None,
-        poolAddress: '0xa5407eae9ba41422680e2e00537571bcc53efbfd',
+        poolAddress: POOLS.curve_sUSD,
         tokens: [TOKENS.DAI, TOKENS.USDC, TOKENS.USDT, TOKENS.sUSD],
     },
-    RenbtcWbtc: {
+    renBTC: {
         exchangeFunctionSelector: CurveFunctionSelectors.exchange,
         sellQuoteFunctionSelector: CurveFunctionSelectors.get_dy,
         buyQuoteFunctionSelector: CurveFunctionSelectors.None,
-        poolAddress: '0x93054188d876f558f4a66b2ef1d97d16edf0895b',
+        poolAddress: POOLS.curve_renBTC,
         tokens: [TOKENS.RenBTC, TOKENS.WBTC],
     },
-    RenbtcWbtcSbtc: {
+    sBTC: {
         exchangeFunctionSelector: CurveFunctionSelectors.exchange,
         sellQuoteFunctionSelector: CurveFunctionSelectors.get_dy,
         buyQuoteFunctionSelector: CurveFunctionSelectors.None,
-        poolAddress: '0x7fc77b5c7614e1533320ea6ddc2eb61fa00a9714',
+        poolAddress: POOLS.curve_sBTC,
         tokens: [TOKENS.RenBTC, TOKENS.WBTC, TOKENS.sBTC],
     },
-    TriPool: {
+    HBTC: {
         exchangeFunctionSelector: CurveFunctionSelectors.exchange,
         sellQuoteFunctionSelector: CurveFunctionSelectors.get_dy,
         buyQuoteFunctionSelector: CurveFunctionSelectors.None,
-        poolAddress: '0xbebc44782c7db0a1a60cb6fe97d0b483032ff1c7',
+        poolAddress: POOLS.curve_HBTC,
+        tokens: [TOKENS.hBTC, TOKENS.WBTC],
+    },
+    TRI: {
```

Comment:
I dont think we actually use these names anywhere, so we may as well use the address as a name?

```suggestion
    [POOLS.curve_TRI]: {
```

Reply:
👍  good point